multiqc/modules/hicpro/hicpro.py

```diff
@@ -44,6 +44,8 @@ def __init__(self):
             data['percent_valid'] = float(data['valid_interaction']) / float(data['total_R1']) * 100.0
             data['Failed_To_Align_Read_R1'] = int(data['total_R1']) - int(data['mapped_R1'])
             data['Failed_To_Align_Read_R2'] = int(data['total_R2']) - int(data['mapped_R2'])
+            if 'valid_pairs_on_target' in data.keys():
```

Comment:
Minor thing, but I would think a "more pythonic" way of writing this would be:

```python
try:
    data['valid_pairs_off_target'] = int(data['valid_interaction']) - int(data['valid_pairs_on_target'])
except KeyError:
    pass
```